src/WebJobs.Script.WebHost/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -85,16 +85,22 @@ public static async Task<JObject> ToFunctionTrigger(this FunctionMetadata functi
             // Read function.json as a JObject
             var functionConfig = await GetFunctionConfig(functionMetadata, functionMetadataFilePath);
 
-            if (functionConfig.TryGetValue("bindings", out JToken value) &&
-                value is JArray)
+            if (functionConfig.TryGetValue("bindings", out JToken value) && value is JArray jArray)
             {
                 // Find the trigger and add functionName to it
-                foreach (JObject binding in (JArray)value)
+                foreach (JObject binding in jArray)
                 {
                     var type = (string)binding["type"];
                     if (type != null && type.EndsWith("Trigger", StringComparison.OrdinalIgnoreCase))
                     {
                         binding.Add("functionName", functionMetadata.Name);
+
+                        string group = functionMetadata.GetFunctionGroup();
```

Comment:
How about we scope this to just Flex consumption? Not needed or used by any other skus.

Reply:
I can add that if we prefer. I left it out because publishing of grouping is not sku dependent, and I wouldn't want anything to be making assumptions on sku based on presence or not of grouping.